### ISSUE TITLE
image link issue

### ISSUE THREAD

**Issue by NsLib** [suan/vim-instant-markdown#63]
I've tried the following code to show image from my hard drive, but it does't work as expected.

```
![image a](images/SPDY-design-arch.png)
![image b](/Users/mdl/vimwiki/html/images/SPDY-design-arch.png)
![image c](file:///Users/mdl/vimwiki/html/images/SPDY-design-arch.png)
![image d](images/reading-notes/arch/posa vol1/chapter2/layer arch.png)
![image e](/Users/mdl/vimwiki/html/images/reading-notes/arch/posa vol1/chapter2/layer arch.png)
```

Maybe add relative image path support?

**Comment by wfaulk** [comment]
You can fake the github workaround into doing this:

    ![image f](/a/raw/b/fileInThisDirectory.png)
    ![image g](/a/raw/b/subdirectory/fileInSubdir.png)

**Comment by tkhm** [comment]
I found why `/a/raw/b/` works on here. https://github.com/suan/instant-markdown-d/blob/18e5dd72ddc8bd0d2ee21731d9d6687551ef8fe6/instant-markdown-d#L93
Thanks, @wfaulk .

**Comment by timbedard** [comment]
Hmmm. I can't get this working using the suggested method. Still no images loading.

**Comment by ashwinvis** [comment]
Fixed in #60
